html/pfappserver/root/src/views/Configuration/switches/_components/TheForm.vue

```diff
@@ -157,6 +157,24 @@
             </div>
           </b-card>
 
+          <b-card v-show="supports(['AccessListBasedEnforcement'])"
+            class="mb-3 pb-0" no-body
+          >
+            <b-card-header>
+              <h4 class="mb-0" v-t="'Role mapping by Access List'"></h4>
+            </b-card-header>
+            <div class="card-body pb-0">
+              <form-group-toggle-access-list-map namespace="AccessListMap"
+                :column-label="$i18n.t('Role by Access List')"
+              />
+
+              <form-group-role-map-access-list v-for="role in roles" :key="`${role}AccessList`" :namespace="`${role}AccessList`"
```

Comment:
Make sure that it clearly states in the admin that putting an ACL here will supersede the ACL defined in the role